File: baekjoon/1977.cpp
#include<stdio.h>

int d(int n) {
    for(int i = 1; i <= 100; i++) {
        if(i * i == n) return 1;
    }
    return 0;
}

int main() {
    int m, n;
    scanf("%d %d", &m, &n);
    int min = 0, sum = 0;

    for(int i = m; i <= n; i++) {
        if(d(i) == 1) {
            if(min == 0) min = i;
            sum += i;
        }
    }
    if(sum == 0) printf("-1\n");
    else printf("%d\n%d", sum, min);
}

File: works/js/3-5/1977.js
let fs = require('fs');

var [M, N] = fs.readFileSync(0).toString().trim().split('\n').map(Number)
// var [M, N] = fs.readFileSync('input.txt').toString().split('\n').map(Number)

var sum = 0 
var minVal = 100000
for(i = 1; i <= 100; i++) {
    var n = i * i 
    if(n >= M && n <= N) {
        sum += n 
        minVal = Math.min(minVal, n) 
    }
}

if(sum == 0) {
    console.log(-1)
}
else {
    console.log(sum)
    console.log(minVal)    
}
